_faqs/faq-12.md

```diff
@@ -1,8 +1,114 @@
+Hence, there is not a single right choice for every application.
+In this comparison we try to give a balanced comparison of the high-level architectural trade-offs other operating systems took and how they compare to RIOT.
+Additionally, we compare the government of the project and the licensing.
 
+##### RIOT
+
+RIOT runs on many different CPU architectures by many different vendors, from [a tiny 8-bit MCU with 2 KiB RAM and 32 KiB flash][riot-arduino-nano] all the way up to high-performance [ARM Cortex M][riot-same54-xpro] and [RISC-V][riot-esp32c3-devkit] MCUs.
+Because RIOT abstracts the hardware differences by providing the same API for the same functionality across this large range of MCUs, projects building upon RIOT are highly portable and therefore resilient to changes in the supply chain.
+
+RIOT is designed to be a general-purpose OS with batteries included.
+It provides a large collection of in-tree modules and out-of-tree packages that seamlessly integrate in your application.
+This allows developers to pick the features they need from a large collection (as long as they can fit on the MCU).
+
+RIOT aims to be easy to learn by using sane defaults, by providing good documentation, and via its helpful community that happily answers questions [in the forum][riot-forum] or in the [matrix chat][riot-matrix].
+However, RIOT does not sacrifice advanced features, productivity or the ability to tinker with settings for more simplicity.
+This makes RIOT suitable for students in teaching, tinkerers in the DIY community, and professionals alike.
+
+RIOT is developed by a diverse, open, and [self-governed][riot-community-process] community.
+Members in the community mostly belong to one (or more) of the following groups: academia, industry, and makers/hobbyists.
+
+RIOT is distributed under a copyleft license and strongly prefers free and open source components over proprietary code and binary blobs.
+Still, RIOT does contain proprietary drivers (some even using binary blobs) where no free alternative is available, e.g. for the ESP WiFi chipset.
+<!-- FIXME: Make FAQ entries linkable and link to "Why LGPL?" -->
+
+##### Vendor Specific SDKs
+
+Most MCU vendors provide an SDK that is tied to and only supports MCUs of that specific vendor.
+The advantage of the vendor SDKs is that they are developed in lock-step with the hardware and expose all features of the hardware early on.
+The RIOT community can only start working on drivers once the MCUs are generally available and more exotic features that are only relevant to niche use cases may never be supported in favor of a cleaner and more portable API in RIOT.
+
+From a governance point of view, RIOT is developed by a self-governed community, while the vendor SDKs are under tight control of the vendor.
+The RIOT community has so for maintained many popular boards well after the vendors declared them obsolete, while vendor SDKs often phase out support for older MCUs sooner.
```

Comment:
```suggestion
The RIOT community has so far maintained many popular boards well after the vendors declared them obsolete, while vendor SDKs often phase out support for older MCUs sooner.
```

or

```suggestion
Because of that, the RIOT community has maintained many popular boards well after the vendors declared them obsolete, while vendor SDKs often phase out support for older MCUs sooner.
```

Or what was meant here? Having trouble parsing this sentence.

Reply:
Nice catch, thx :)